.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: java
+jdk:
+  - openjdk8
+services:
+  - docker
+
+branches:
+  only:
+    - "/^v[0-9]+\\.[0-9]+\\.[0-9]+.*$/"
+
+install: true
+
+before_script:
+  - sudo chown -R $USER:$GROUP $TRAVIS_BUILD_DIR
+
+script:
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ] && [[ "$TRAVIS_BRANCH" =~ ^v[0-9]+\.[0-9]+\.[0-9]+.*$ ]]; then cp ./src/devops/release.sh . && ./release.sh; fi
+  - export TAG=$TRAVIS_BRANCH
+  - export IMAGE_NAME='hygieiadoc/githubscmcollector'
```

Comment:
imagename should be terraforminfracollector

Reply:
Done